### PR TITLE
Clean up melee.json take 2 [updated 4/11]

### PR DESCRIPTION
**Pull request by Cyrano7**
This time more organized.

### CODE REVIEW

File: data/json/items/melee.json [hunk 1]
```diff
@@ -387,8 +369,7 @@
         "techniques" : ["WBLOCK_1"],
         "volume" : 5,
         "bashing" : 4,
-        "price" : 2400,
-        "price_postapoc": 0
```
Comment (Coolthulhu) [inline data/json/items/melee.json]: Don't delete `"price_postapoc" : 0` from useless items. It defaults to `"price"`, not to 0.

Comment (Cyrano7) [inline data/json/items/melee.json]: Ya I didin't delete all of them. Just where I thought it made sense.

Comment (Coolthulhu) [inline data/json/items/melee.json]: For things like brooms and two-by-fours it doesn't really make sense that people would keep buying them when they can just smash nearby furniture. They're worth as much as rocks or scrap metal.

Comment (Cyrano7) [inline data/json/items/melee.json]: True I'll fix it when I get home. I'm going to be away for the next week.

**Comment by Cyrano7** [comment]
@Coolthulhu I added back the "price_postapoc": 0 finally.

**Pull request merged**